libs/api-implementation/models/test-run/test-run.schema.ts

```diff
@@ -0,0 +1,39 @@
+import { Schema } from 'mongoose';
+
+const TestRunSchema = new Schema({
+  githubNumber: {
+    type: String,
+    required: true,
+  },
+
+  type: {
+    type: String,
+    enum: ['Commit', 'PullRequest'],
+    required: true,
+  },
```

Comment:
Of course, in terms of reusability, it would be better to create one enum object and reuse it elsewhere. As far as I've found on mongoose docs, the enums are created as an array of strings, which are not the case for the typescript enums. 

I think it is worth to test it, is it compatible with the ts enums (again, didn't found any info on docs).

Reply:
Oh! I see. Do you think [this stackoverflow](https://stackoverflow.com/questions/44290887/using-typescript-enum-with-mongoose-schema) answer would be helpful to you?

Reply:
Yeap, doing that way is a little bit hacky, but will do the job I guess... I'll add the details to the issues